samples/advanced-nodejs/routes/requests.js

```diff
@@ -5,21 +5,13 @@ var express = require('express');
 var router = express.Router();
 var request = require('request');
 
-// Insert your Azure subscription key here
-var cogSvcsSubscriptionKey = '';
-
-// The location associated with the Immersive Reader resource.
-// The following are valid values for the region:
```

Comment:
I changed it to use 'endpoint' instead of region, because the endpoint is what the user sees on https://azure.microsoft.com/try/cognitive-services/my-apis/.